tools/preflight/preflight.go

```diff
@@ -728,6 +726,8 @@ func (o *Run) createSourcePVC(ctx context.Context, nameSuffix string,
 	pvc = createVolumeSnapshotPVCSpec(o, SourcePvcNamePrefix+nameSuffix, nameSuffix)
 	pvc, err = k8sClient.CoreV1().PersistentVolumeClaims(o.Namespace).Create(ctx, pvc, metav1.CreateOptions{})
 	if err != nil {
+		srcPVCYaml := objToYAML(pvc)
+		o.Logger.Warnf("failed to create source PVC. PVC YAML: %s\n", string(srcPVCYaml))
```

Comment:
1. Don't store the yaml data in a separate variable. Use the return value of func `objToYAML(pvc)` directly in the log statement. Do this wherever applicable.
2. Print the yaml on new line after colon.
3. yaml word should be lowercase - `. PVC yaml: `
4. For warning statements follow punctuation rules. Log stmt can start with a capital letter.